File: backend/app/src/utils/id.py
from typing import Any


/**
 * Generates a unique identifier using a combination of timestamp and random string
 * @returns A unique string identifier
 */
function generateUniqueId(): str {
  const timestamp = Date.now()
  const random = Math.random().toString(36).substring(2, 15)
  return `${timestamp}_${random}`
} 

File: src/utils/id.ts
/**
 * Generates a unique identifier using a combination of timestamp and random string
 * @returns A unique string identifier
 */
export function generateUniqueId(): string {
  const timestamp = Date.now();
  const random = Math.random().toString(36).substring(2, 15);
  return `${timestamp}_${random}`;
} 